### ISSUE TITLE
Events grouped together

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#661]
New site

It would be useful to be able to display all the events to do with a particular thing, all grouped together via some sort of filter. For example, there is a range of events to do with the HH Music festival. If it was possible to filter according to a grouping then we could see an event calendar for a series i.e. the festival event calendar. Ideally when the event was created it could be part of "something bigger", a box to tick etc.

**Comment by djwesto** [comment]
Hi @GilesGibson,

Do you think you could possibly give us a range of examples of all the
sorts of things you'd like to see grouped under a "Something bigger"
grouping and where you'd like that to appear?

You're talking about building something that would be pretty complicated,
organisationally-speaking (i.e. not even from a tech standpoint, how on
earth would you control and monitor this? Is it something you'd want
authors to specify and create themselves or would only editors be able to
create the "Something bigger"?)

Lots to think about.

When we're wireframing the new Open Local wireframes, we could look at
integrating this – would it be something you'd want to see on the landing
page of the Events section or somewhere in its own section (i.e. on this
"Visit Herne Hill" TimeOut-style section we've been discussing recently?)

C

---

_Chris Wells_\* Co-founder & Creative Director*
Folk Labs Ltd           www.folklabs.com           linkedin
http://lnkd.in/uZxawZ          twitter @djwesto
http://www.twitter.com/djwesto          mobile +44 (0)7786 221 654

**Comment by GilesGibson** [comment]
I have just moderated about 20 events, all part of the Herne Hill Music Festival. A user may want to filter every event for the festival so that they can see what is on, where and when. The HH Free Film Festival was the same, about 4 major events plus 12 or so in the fringe events. If it was possible to have a block somewhere on the home page that we could put up that just had "Herne Hill Music Festival - events guide", a user clicked on it and it showed a filtered list of all the events for the music festival then that would be helpful. 

Having it as part of the Time Out guide would be good too.

I suppose that ideally the event manager would want to create a filter category - Music Festival, or "Film Festival or festival of Egyptian stretching and then click their event to be part of it. The filter option would then be show to users so they could filter events they see, a bit like "child friendly" or "outdoor".If we could have a way of having a block on the front page to be able to go to a place on the site and display a pre-determined filtered view it would be good. It could show all the service listers who were "market stall" or "herne hill music festival event".

**Comment by djwesto** [comment]
@pmackay I'm thinking about a slight alternative to @GilesGibson's idea which would be to create sub-terms under any Categories this would be relevant to, so perhaps we might put the Film Festival in under **Music, Film, Dance**...

Let's discuss next week.

The design of the Blocks Giles is asking for are something else – they'd probably need a special teaser style.

**Comment by GilesGibson** [comment]
having it under film festival dance would be as expected. What would be good would be to have it as a dedicated title. makes it easier for us to send people there via twitter and so forth, a pre-filtered event search or a pre-filtered service listing search.

**Comment by djwesto** [comment]
So, you're basically saying that (in essence) you agree that it would make
sense to place these sort of series' of events (i.e. the Film Festival)
under an appropriate category?

If so, it would be really helpful if you could assemble a list of all such
"series" or event groupings that you can currently think of. Add them as a
bullet list in a comment under this issue and we can take the first steps
of figuring out whether they work under the existing category taxonomy.

I think @pmackay will likely suggest this is going to be a bit tricky to
deliver but a pre-filtered search would (I think) allow you to send around
a URL which would take you to a listing page showing just the content under
that specific category – this is the easiest way I can think of to do
this...

Cheers

---

_Chris Wells_\* Co-founder & Creative Director*
Folk Labs Ltd           www.folklabs.com           linkedin
http://lnkd.in/uZxawZ          twitter @djwesto
http://www.twitter.com/djwesto          mobile +44 (0)7786 221 654

**Comment by GilesGibson** [comment]
The two events that would warrant a generic group listing would have been the Herne Hill Free Film Festival and the Herne Hill Music Festival. The service listings that would naturally group together would be "Market Stall Holders" and maybe a new category of "Herne Hill Retail trader".

**Comment by djwesto** [comment]
You might remember that we set up "Local economy" as a category to represent all local traders and other business people...

**Comment by GilesGibson** [comment]
yes, but that is a little too generic. With the plans to put some local traders on the google places map it is also important to be able to list all the local retail traders (i.e. shops) over and above the yoga teachers etc.

**Comment by djwesto** [comment]
I'm not entirely sure I agree the best way to do this is to add yet another category, @GilesGibson... The listing is already getting quite unwieldy... Perhaps we need to find a way to split it out so that only some category terms appear in certain places where they're relevant, but (for now), might it make more sense to swap around the wording of the terms slightly? I'm specifically thinking that **Market stall traders** and **Retailers** could live together in one category, quite comfortably... No?

**Comment by GilesGibson** [comment]
No, we need to keep retail shops separate from market stalls. There is a group for the market, all the news and features about the market. we need to be able to show easily all the market stalls that are at the market - a simple filter would show that. Ditto, all the local shops would need to be shown in a list (or better) as per the funding potentially available via Tjames Water and our discussions with you about getting them listed on google places.

**Comment by djwesto** [comment]
So, for these sorts of listings which filter out individual types of orgs (in this instance "local retailers") are you saying you want to be able to write some specific content on this retailers listing page?

Or would you be happy if it was purely just a listing on the /organisations page we'll be building which has filtered out everything except local retailers? Here you can see a page showing a filtered list just showing local builders: 

> http://www.hernehill.org.uk/services?f[0]=field_ol_business_category%3A6938

I take it that would suffice for now?

**Comment by GilesGibson** [comment]
spot on, for now that would be fine. To get two gold stars, having the one filter that was used to be displayed as a page title would be awesome ie. "Herne Hill Market Traders". 

Bet you want to know what would get you three gold stars....

**Comment by pmackay** [comment]
So just checking back on this as I was away when it was raised, you've combined two issues about grouping events and categorising market traders. Seems to be separate threads.

On the events issue, I reckon the best way to get a combined listing would be one of:
- create a group/organisation and add them all to that?
- create a news article and associate events with that, which could be done if we build the general "related content" field that's been discussed before.
- use a dedicated tag, e.g. `hhfilmfest2015`

Not sure using categories works cos they are too general and will fill up with lots of stuff.
